tableaudocumentapi/column.py

```diff
@@ -0,0 +1,62 @@
+import functools
+
+_ATTRIBUTES = [
+    'name',
+    'caption',
+    'datatype',
+    'role',
+    'type',
+    'alias',
+    'calculation',
+]
+
+_METADATA_ATTRIBUTES = [
+    'aggregation',
+]
+
+
+def _find_metadata_record(record, attrib):
+    element = record.find('.//{}'.format(attrib))
+    if element is None:
+        return None
+    return element.text
+
+
+class Column(object):
```

Comment:
Do we think columns are different than fields (honest question)?

Our literature tends to refer to these as "Fields" -- but I could see that being a higher level object, depending on what you're thinking.